src/coreclr/jit/lower.cpp

```diff
@@ -1911,6 +1981,7 @@ void Lowering::LowerCall(GenTree* node)
     JITDUMP("lowering call (after):\n");
     DISPTREERANGE(BlockRange(), call);
     JITDUMP("\n");
+    return nullptr;
```

Comment:
Couldn't this return `call->gtNext` and then we could avoid all the special-case checking for nullptr?

Reply:
I can't return `call->gtNext` here becuase `gtNext` can be nullptr itself so then it won't be clear whether this method made any changes or not. In fact, I did it initally and hit an assert somewhere